client/src/components/CheckboxSelector.js

```diff
@@ -0,0 +1,37 @@
+import './CheckboxSelector.scss';
+
+export const FancyCheckboxSelector = ({items, callbackToggle}) =>
+  _.map(items,
+    (item, key) => <FancyCheckbox key={key} callbackToggle={callbackToggle} {...item}/>
+  );
```

Comment:
I think it would make sense for this component to also take a `legend` prop, and to return something like
```
<fieldset>
  <legend>{legend}</legend>
  { _.map(
    items,
    (item, key) => <FancyCheckbox key={key} callbackToggle={callbackToggle} {...item}/>
  ) }
</fieldset>
```

Maybe with a `WellList` style border around it by default. If we ever want a non-default style somewhere we can import `FancyCheckbox` and build it as needed.

Reply:
Hmm, bootstrap has some pretty heavy styling on `legend`'s. Might need to add a `legend-unstyled` utility class in `extended_bootstrap_css/bootstrap-wet-fixes-exstensions.scss` to reset all of that styling.